src/test/java/com/fasterxml/jackson/databind/format/MapEntryFormatTest.java

```diff
@@ -177,7 +177,6 @@ public void testDefaultShapeOverride() throws Exception
                     cfg.setFormat(JsonFormat.Value.forShape(JsonFormat.Shape.OBJECT)))
                 .build();
         Map.Entry<String,String> input = new BeanWithMapEntry("foo", "bar").entry;
-        assertEquals(a2q("{'key':'foo','value':'bar'}"),
-                mapper.writeValueAsString(input));
+        assertTrue(mapper.writeValueAsString(input).equals(a2q("{'key':'foo','value':'bar'}")) || mapper.writeValueAsString(input).equals(a2q("{'value':'bar','key':'foo'}")));
```

Comment:
Ideally would read into `Map` or `JsonNode`, but with just 2 entries this is ok too.